mantidproject/mantidimaging
# Asks for confirmation when closing 180deg proj stack, and properly clears it from parent images if user OK the close

To test:
- Load a dataset with a 180deg image
- Try to close it: you should get a confirmation prompt
  - Check that cancel/closing it via X doesn't close the stack
- On OK it should close
- Open reconstruction window and try to correlate - it should calculate it OK and not show an error

Fixes #679 
Fixes #680
